src/main/kotlin/com/atlassian/performance/tools/report/api/judge/ShiftedDistributionRegressionTestProvider.kt

```diff
@@ -0,0 +1,19 @@
+package com.atlassian.performance.tools.report.api.judge
+
+import com.atlassian.performance.tools.report.api.ShiftedDistributionRegressionTest
+
+open class ShiftedDistributionRegressionTestProvider {
```

Comment:
Suggestion: If we want to make polymorphic solution out of it I'd suggest to think of more generic abstraction of "regression test" or maybe more like "regression judge".

I see how this specific abstraction we are looking for is comparing 2 sets of results. The `mwAlpha` and `ksAlpha` are more of parameters that define variants of this judge and are implementation specific. Perhaps we could pass those alphas when building this regression test/judge provider or even just a regression judge (without provider)?